### ISSUE TITLE
Tox tests over different reason

### ISSUE THREAD

**Issue by ad-m** [pydanny/cookiecutter-djangopackage#49]
Hello,

Thanks for great repo. I started up my project yet ( https://github.com/ad-m/django-teryt-tree ) using your starter.

I am would like why you don't provide tests over different version of django for travis or pull requests is well come? Is there any reason to don't tests by tox&travis over different version of django package?

Greetings,
Adam Dobrawy

**Comment by ad-m** [comment]
I note patch to achieve it on cooked package:
```diff
diff --git a/.travis.yml b/.travis.yml
index 0121a81..722722f 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -2,19 +2,25 @@
 
 language: python
 
-python:
-  - "3.4"
-  - "3.3"
-  - "2.7"
+env:
+  - TOX_ENV=py27-dj17
+  - TOX_ENV=py33-dj17
+  - TOX_ENV=py34-dj17
+  - TOX_ENV=py27-dj18
+  - TOX_ENV=py33-dj18
+  - TOX_ENV=py34-dj18
+  - TOX_ENV=py27-djmaster
+  - TOX_ENV=py33-djmaster
+  - TOX_ENV=py34-djmaster
 
-before_install:
-  - pip install codecov
+matrix:
+  allow_failures:
+    - env: TOX_ENV=py27-djmaster
+    - env: TOX_ENV=py33-djmaster
+    - env: TOX_ENV=py34-djmaster
 
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -r requirements-test.txt
+before_install:
+  - pip install coveralls tox
 
 # command to run tests using coverage, e.g. python setup.py test
-script: coverage run --source teryt_tree runtests.py
-
-after_success:
-  - codecov
+script: tox -e $TOX_ENV
diff --git a/tox.ini b/tox.ini
index b853f7b..c3363e3 100644
--- a/tox.ini
+++ b/tox.ini
@@ -1,9 +1,20 @@
 [tox]
-envlist = py27, py33, py34
+envlist = py{27,33,34}-dj{17,18,master}
+basepython =
+    py27: python2.7
+    py33: python3.3
+    py34: python3.4
 
 [testenv]
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/teryt_tree
-commands = python runtests.py
+commands = 
+    coverage run --source teryt_tree runtests.py
+    py27-dj18: coveralls
+
 deps =
+    dj18: Django>=1.8,<1.9
+    dj17: Django>=1.7,<1.8
+    djmaster: https://github.com/django/django/archive/master.tar.gz
     -r{toxinidir}/requirements-test.txt
```

**Comment by pydanny** [comment]
@ad-m I would love this as a pull request!

**Comment by ad-m** [comment]
If you use hitch you tests project over different python version:
> Although it says python 3.4 below, your code will be tested with the version of python you specified in your tests (assuming you are testing python code at all). python 3.4 will only be used to run the bootstrap script.
https://hitchtest.readthedocs.org/en/latest/howto/continuous_integration.html
I don't know how integrate this with hitch. If you prefer tox than hitch - I believe I can create pull requests.

**Comment by pydanny** [comment]
This was addressed in #80
